apps/site/src/views/Resources/sections/Landing/Landing.tsx

```diff
@@ -1,9 +1,12 @@
+import Image from "next/image";
+import resourcesTitle from "@/assets/images/resources-title.svg";
+
 import styles from "./Landing.module.scss";
 
 function Landing() {
 	return (
 		<div className={styles.landing}>
-			<h1>Resources</h1>
```

Comment:
For document accessibility, it would be good for the page to still have a main heading element which we can make [visually hidden](https://getbootstrap.com/docs/5.3/helpers/visually-hidden/)